src/Symfony/Component/Mailer/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * added the `mailer` monolog channel and set it on all transport definitions
+ * Add support for `SESSION_TOKEN` for Async AWS Mailer transport
```

Comment:
this is not totally accurate. this PR does not add support for the end variable `SESSION_TOKEN`, but adds a new option in DSN.